src/target/riscv/riscv.c

```diff
@@ -2282,17 +2282,19 @@ static int riscv_address_translate(struct target *target,
 static int riscv_virt2phys_v(struct target *target, target_addr_t virtual, target_addr_t *physical)
 {
 	riscv_reg_t vsatp;
-	if (riscv_get_register(target, &vsatp, GDB_REGNO_VSATP) != ERROR_OK) {
-		LOG_TARGET_ERROR(target, "Failed to read vsatp register.");
+	int result = riscv_get_register(target, &vsatp, GDB_REGNO_VSATP);
+	if (result != ERROR_OK) {
+		LOG_TARGET_ERROR(target, "Failed to read vsatp register. Error: %d", result);
 		return ERROR_FAIL;
```

Comment:
```suggestion
		return result;
```

Reply:
After CS discussion we decided not to print/return error code here, because it looks like useless for users.

Reply:
That is fine.